submissions/antonkottans/js-dom/script.js

```diff
@@ -0,0 +1,136 @@
+        Отлично подходит для приготовления диетических блюд. При запекании нужно добавлять
+        масло, а при жарке нарезать толстыми стейками, т.к. рыба сама по себе сухая.
+    </section>
+</article>`,
+            `		<article>
+    <h2  class="content-heading">Щука</h2>
+    <img class="content-img" alt="pike on grass" src="./img/pike.jpg" />
+    <section>
+        <h3  class="content-heading">Общее описание рыбы</h3>
+        Живёт обычно в прибрежной зоне, в водных зарослях, в непроточных или слабопроточных
+        водах. Тело торпедовидное, голова большая, пасть широкая. Основу питания щуки
+        составляют представители различных видов мелких рыб.
+    </section>
+    <section>
+        <h3  class="content-heading">Приготовление</h3>
+        Обычно рыба имеет сильный привкус, который не всем по вкусу. Что избавиться от него
+        лучше запекать или тушить.
+    </section>
+</article>`,
+            `
+    <article>
+        <h2  class="content-heading">Сом</h2>
+        <img class="content-img" src="./img/catfish.jpg" alt="catfish">
+        <section>
+            <h3  class="content-heading">Общее описание рыбы</h3>
+            Обитает в реках и озёрах. В передней части тело речного сома имеет округлые формы, а
+            с продвижением к хвосту становится сжатым с обоих боков и переходит в сильный хвост.
+        </section>
+        <section>
+            <h3  class="content-heading">Приготовление</h3>
+            Умеренно жирная рыба. Из-за толстой кожи, можно готовить как угодно и на гриле со
+            сгоревшей в уголь кожей тоже норм, т.к. её далеко не все едят.
+        </section>
+    </article>`,
+            `		<article>
+    <h2  class="content-heading">Карась</h2>
+    <a title="jag asså, Public domain, via Wikimedia Commons" href="https://commons.wikimedia.org/wiki/File:Carassius_carassius_1.jpg"><img class="content-img" alt="Carassius carassius 1" src="https://upload.wikimedia.org/wikipedia/commons/thumb/6/6c/Carassius_carassius_1.jpg/512px-Carassius_carassius_1.jpg"></a>
+    <section>
+        <h3  class="content-heading">Общее описание рыбы</h3>
+        Питаются растительностью, мелкими беспозвоночными, зоопланктоном, зообентосом и
+        детритом. Обитают исключительно в болотистых и низменных озёрах и реках. Спинной
+        плавник длинный, глоточные зубы однорядные. Тело высокое с толстой спиной, умеренно
+        сжатое с боков. Чешуя крупная и гладкая на ощупь.
+    </section>
+    <section>
+        <h3  class="content-heading">Приготовление</h3>
+        Имеет сильный приятный сладковатый вкус и ооочень много костей. В домашних условия
+        подают обычно жареным с крестообразными надрезами. Также очень вкусный получается
+        запеченый с майонезом, сметаной и зеленью.
+    </section>
+</article>`,
+        ];
+        let cashedArticles = [];
+        let fishNames = ["carp", "grassCarp", "pike", "catfish", "carassius"];
+        fishNames.forEach((fishName) =>
```

Comment:
Interesting choice, but you can't you just do it inside a single cycle? And your `cachedArticles` array seems like it can store just objects. This way, it'll be more clear with syntax like `contentItem.element` or `contentItem.fishName` than `contentItem[0]`/`contentItem[1]`.

Reply:
You can simlify it and avain unnecesary calculations
`const cashedArticles  =  fishNames.map((fishName) =>....`
Combine logic from two `forEach` array iteration in one `map` using Artem's advice.

Reply:
i used one forEach instead of map, i hope its not a mistake